src/postgraphql/http/createPostGraphQLHttpRequestHandler.d.ts

```diff
@@ -48,6 +49,10 @@ export default function createPostGraphQLHttpRequestHandler (config: {
   // value.
   jwtSecret?: string,
 
+  // Custom options given to jwt.verify function. When no custom options were given
+  // the default audience will be set to 'postgraphql'
+  jwtOptions?: VerifyOptions,
```

Comment:
It would be more accurate to say when no `audience` property is given. Unless I’m mistaken about the implementation? 😊